src/service/Domain/FeatureFilters/RulesEngineFilter.cs

```diff
@@ -44,7 +49,7 @@ public async Task<bool> EvaluateAsync(FeatureFilterEvaluationContext context)
 
             try
             {
-                FilterSettings filterSettings = context.Parameters.Get<FilterSettings>() ?? new FilterSettings();
+                FilterSettings filterSettings = (FilterSettings)context.Settings; //context.Parameters.Get<FilterSettings>() ?? new FilterSettings();

```

Comment:
nit: can we remove the commented if not required.

Reply:
done